processing.go

```diff
@@ -91,6 +91,10 @@ func Process(mon *Monitor) {
 	//Start all the workers
 	for i := 0; i < workers; i++ {
 		go func() {
+			for _, scannerName := range orderedScanners {
+				scanner := *scanners[scannerName]
+				scanner.InitPerSender(i)
```

Comment:
The intent is to call this once for each i in [0, workers), right? If so, the outer for loop may be a problem --
 you would need to increment the shared counter inside the goroutine, right?

I confirmed by running this with a log method in InitPerSender, that it does indeed get called 1000 times, but, say, ten times for senderID=1000, but not at all for senderID=998.

Reply:
Yeah, you'll need to pass `i` as an argument to the anonymous function:

```
for int i: =0; ....
    go func(i int) {
        for .....
    }(i) 
```

Reply:
Good catch @justinbastress!

Reply:
Pushed a change, thanks @justinbastress!